service/history/execution/context_util.go

```diff
@@ -155,5 +155,15 @@ func emitWorkflowCompletionStats(
 			tag.WorkflowRunID(runID),
 			tag.WorkflowDomainName(domainName),
 		)
+	case types.EventTypeWorkflowExecutionContinuedAsNew:
+		scope.IncCounter(metrics.WorkflowContinuedAsNew)
```

Comment:
Also add a log here?

Reply:
I'm happy to if you think we should have one. It felt like WorkflowContinuedAsNew was normal enough that it didn't warrant a log (similar to the Completed / Cancelled / Failed cases), but perhaps they should all log?